packages/esm-patient-orders-app/translations/en.json

```diff
@@ -18,15 +17,10 @@
   "dose": "Dose",
   "editResults": "Edit results",
   "endDate": "End date",
-  "enterTestResults": "Enter test results",
   "error": "Error",
   "errorCancellingOrder": "Error cancelling order",
-  "errorSavingLabResults": "Error saving lab results",
   "indication": "Indication",
   "launchOrderBasket": "Launch order basket",
-  "loading": "Loading",
-  "loadingInitialValues": "Loading initial values",
-  "loadingTestDetails": "Loading test details",
```

Comment:
In this case, where we're moving strings between packages, it is important that we retain the existing translations in non-English languages. Unfortunately, the only way I know to do that is some tedious copy-and-paste work, but this would be the rare instance were manually updating the translation files is for the best.